diff --git a/node_modules/h3-js/dist/browser/h3-js.es.js b/node_modules/h3-js/dist/browser/h3-js.es.js
index 730ca53..0cfb04d 100644
--- a/node_modules/h3-js/dist/browser/h3-js.es.js
+++ b/node_modules/h3-js/dist/browser/h3-js.es.js
@@ -257,7 +257,7 @@ var libh3 = function (libh3) {
   function stringToUTF8(str, outPtr, maxBytesToWrite) {
     return stringToUTF8Array(str, HEAPU8, outPtr, maxBytesToWrite);
   }
-  var UTF16Decoder = typeof TextDecoder !== "undefined" ? new TextDecoder("utf-16le") : undefined;
+  var UTF16Decoder = undefined; // patched for Hermes (no utf-16le TextDecoder)
   function writeArrayToMemory(array, buffer) {
     HEAP8.set(array, buffer);
   }
diff --git a/node_modules/h3-js/dist/browser/h3-js.js b/node_modules/h3-js/dist/browser/h3-js.js
index ca2448a..39bf1c5 100644
--- a/node_modules/h3-js/dist/browser/h3-js.js
+++ b/node_modules/h3-js/dist/browser/h3-js.js
@@ -257,7 +257,7 @@ var libh3 = function (libh3) {
   function stringToUTF8(str, outPtr, maxBytesToWrite) {
     return stringToUTF8Array(str, HEAPU8, outPtr, maxBytesToWrite);
   }
-  var UTF16Decoder = typeof TextDecoder !== "undefined" ? new TextDecoder("utf-16le") : undefined;
+  var UTF16Decoder = undefined; // patched for Hermes (no utf-16le TextDecoder)
   function writeArrayToMemory(array, buffer) {
     HEAP8.set(array, buffer);
   }
diff --git a/node_modules/h3-js/dist/h3-js.es.js b/node_modules/h3-js/dist/h3-js.es.js
index a416b7d..6e6d243 100644
--- a/node_modules/h3-js/dist/h3-js.es.js
+++ b/node_modules/h3-js/dist/h3-js.es.js
@@ -365,7 +365,7 @@ var libh3 = function (libh3) {
   function stringToUTF8(str, outPtr, maxBytesToWrite) {
     return stringToUTF8Array(str, HEAPU8, outPtr, maxBytesToWrite);
   }
-  var UTF16Decoder = typeof TextDecoder !== "undefined" ? new TextDecoder("utf-16le") : undefined;
+  var UTF16Decoder = undefined; // patched for Hermes (no utf-16le TextDecoder)
   function writeArrayToMemory(array, buffer) {
     HEAP8.set(array, buffer);
   }
diff --git a/node_modules/h3-js/dist/h3-js.js b/node_modules/h3-js/dist/h3-js.js
index e70b2f1..eb25b22 100644
--- a/node_modules/h3-js/dist/h3-js.js
+++ b/node_modules/h3-js/dist/h3-js.js
@@ -365,7 +365,7 @@ var libh3 = function (libh3) {
   function stringToUTF8(str, outPtr, maxBytesToWrite) {
     return stringToUTF8Array(str, HEAPU8, outPtr, maxBytesToWrite);
   }
-  var UTF16Decoder = typeof TextDecoder !== "undefined" ? new TextDecoder("utf-16le") : undefined;
+  var UTF16Decoder = undefined; // patched for Hermes (no utf-16le TextDecoder)
   function writeArrayToMemory(array, buffer) {
     HEAP8.set(array, buffer);
   }
